index.md

```diff
@@ -306,7 +257,7 @@ One METS file in the root of the package references all metadata and data files
 Even though the number suffix of the folders `rep-001` and `rep-002` of the
 example shown in Figure [3](#fig3) suggests an order of representations, there
 are no requirements regarding the naming of folders containing the
-representations.  The order of representations and the relations between them is
+representations. The order of representations and the relations between them is
 defined by the structural and preservation metadata. The `representations`
 folder is mandatory, even for IPs which contain only one representation.
```

Comment:
This is a further restriction on the CSIP which only makes the representations folder a SHOULD, not a must.

Reply:
This was a leftover from the previous AIP format. Simply wrong, I removed it.